docs/documentation/quartz-3.x/how-tos/one-off-job.md

```diff
@@ -0,0 +1,44 @@
+# One-Off Job
+
+You can run the simplest job this way.
+
+:::tip
+
+Misfire Mode: Smart
+
+:::
+
+```csharp
+public Task DoSomething(IScheduler schedule)
+{
+    var job = JobBuilder.Create<AnExampleJob>()
+                        .WithIdentity("name", "group")
+                        .Build();
+
+    var trigger = TriggerBuilder.Create()
```

Comment:
If this is only about triggering a job, maybe `IScheduler.TriggerJob` should also be mentioned as that's the easiest option. in this example you have paired a job so this will also work job hasn't been registered yet. 

For `ISchduler.TriggerJob` comes into play whether job has `Durable` flag set - can it be dangling without triggers - otherwise Quartz would remove it as orphaned. Of course a job having triggers like nightly and calling `TriggerJob` would just trigger it "out-of-schedule".